graphql/task_logs_resolver.go

```diff
@@ -177,6 +179,11 @@ func (r *taskLogsResolver) SystemLogs(ctx context.Context, obj *TaskLogs) ([]*ap
 
 func (r *taskLogsResolver) TaskLogs(ctx context.Context, obj *TaskLogs) ([]*apimodels.LogMessage, error) {
 	const logMessageCount = 100
+	const notFoundMessage = "failed to fetch logs with resp '404 Not Found'"
+	task, taskErr := task.FindOneId(obj.TaskID)
+	if taskErr != nil {
```

Comment:
We should consider executions when fetching tasks. This will return no logs for a task with 2 executions when viewing the first execution.